src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -1911,7 +1911,7 @@ public void autopersist(@Nonnull FlowNode n) throws IOException {
         }
 
         @Override public void addContents(Container container) {
-            container.add(new Content("nodes/master/pipeline-timings.txt") {
```

Comment:
This does not break any tests, but may require updates to documentation?

Reply:
This is referring to the _master node_ not the controller, so the replacement should be `built-in` instead of `controller`.